doc/source/examples/generative_design_ex/00-generative_design_model_build.py

```diff
@@ -0,0 +1,176 @@
+
+print(len(project.data()))
+print(project.data())
+
+
+###############################################################################
+# Add the training data to the current project:
+#
+# - | If the training data has never been uploaded to the server,
+#   | use the following script:
+
+for geometry_data_name in os.listdir(my_dataset_path):
+    geometry_data_file = os.path.join(my_dataset_path, geometry_data_name)
+    try:
+        td = client.training_data.create_from_file(file=geometry_data_file, project=project)
+        print(f"Uploaded {geometry_data_name} -> ID: {td.id}")
+    except Exception as e:
+        print(f"FAILED to upload {geometry_data_name}: {e}")
+
+for data in project.data():
+    print(data.name)
+
+
+###############################################################################
+# - If the training data already exists on the server, do as follows:
+#
+# Step 1. Get the project by name:
+
+project_with_TD = client.projects.get(name="new-GeomAI-project")
+
+###############################################################################
+# Step 2. Get all training data in that project:
+
+data_items = project_with_TD.data()
+
+###############################################################################
+# Step 3. Print the desired number of training data to add:
+# (5 in this example)
+
+for td in data_items[:5]:
+    print(f"{td.id}: {td.name}")
+
+training_data_list = data_items[:5]
+
+###############################################################################
+# Step 4. Add each data item to the current project:
+
+for td in training_data_list:
+    try:
+        td.add_to_project(project)
+        print(f"✅ Added {f'{td.id}: {td.name}'} to current project.")
+    except Exception as e:
+        print(f"❌ Failed to add {f'{td.id}: {td.name}'}: {e}")
+
+
+###############################################################################
+# Remove data from the project
+# ----------------------------
+# Use the script below to remove data from the project:
+
+for data in project.data():
+    try:
+        data.remove_from_project(project)
+        print(f"Removed: {data.name}")
+    except Exception as e:
+        print(f"Could not remove {data.name}: {e}")
+
+
+###############################################################################
+# Check the result of the removal by displaying the number of data in the
+# current project and listing them all:
+
+print(len(project.data()))
+print(project.data())
+
+
+###############################################################################
+# Set a model configuration for the newly created project and build the model
+# ---------------------------------------------------------------------------
+
+
+###############################################################################
+# Create a model configuration:
+
+configuration = GeomAIModelConfiguration(
+    nb_epochs=2,  # build_preset: 'debug', 'short', 'default' or 'long'
```

Comment:
```suggestion
    nb_epochs=2,  
    # or build_preset: 'debug', 'short', 'default' or 'long'
```
This comment on the same line is confusing IMO, I'd think it describes `nb_epochs` but it doesn't make sense in that context.